src/main/java/step3/domain/CarRacingRapScore.java

```diff
@@ -0,0 +1,23 @@
+package step3.domain;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class CarRacingRapScore {
+
+    private List<Car> cars;
+
+    public CarRacingRapScore(List<Car> cars) {
+        this.cars = cars.stream().map(Car::clone).collect(Collectors.toList());
+    }
+
+    public List<Car> getCars() {
+        return cars;
+    }
+
+    public List<Car> findBestScoreCars() {
+        int bestScore = cars.stream().map(Car::getPosition).max(Integer::compareTo).get();
```

Comment:
```suggestion
        int bestScore = cars.stream()
                .mapToInt(Car::getPosition)
                .max()
                .orElseThrow(IllegalArgumentException::new);
```
Optional 객체는 주의해서 사용하는것을 권장드려요.
아래 문서를 통해 Optional에 대해 조금더 알아보세요 ! 
http://homoefficio.github.io/2019/10/03/Java-Optional-%EB%B0%94%EB%A5%B4%EA%B2%8C-%EC%93%B0%EA%B8%B0/